### DISCUSSION TITLE
No auto multi-line height adjustment?

### DISCUSSION THREAD

**Discussion by thibaultmol** [JulianPrieber/littlelink-custom#407] [Feature requests]
I kind of assumed that if I put in text for the button that is longer than one line, it would automatically increase the height of the button. But it doesn't?

![image](https://user-images.githubusercontent.com/1010226/233699106-0a510d87-953e-48fd-8fd4-197ab0c4fd5a.png)

**Comment by JulianPrieber** [comment]
It does not. This is more or less a deliberate choice.
We use the same styling as our parent project for the buttons: https://github.com/sethcottle/littlelink

**Comment by thibaultmol** [comment]
Ah, fair decision then tbh. I'll check upstream if someone has made an issue about the same thing

**Comment by thibaultmol** [comment]
see https://github.com/sethcottle/littlelink/issues/90

**Comment by JulianPrieber** [comment]
brilliant

**Comment by JulianPrieber** [comment]
You might have better luck with some themes. I don't think they support multi line text, but they do have a wider button width.

Not sure how this would work on mobile.

**Comment by JulianPrieber** [comment]
https://github.com/LinkStackOrg/LinkStack/issues/594

**Reply by beedell-roke** [reply]
> https://github.com/orgs/LinkStackOrg/discussions/407#discussioncomment-7240393

@JulianPrieber, I don't understand the outcome of this issue. https://github.com/sethcottle/littlelink/issues/90#issuecomment-1529911126 states that this was remediated, yet this downstream project does not appear to be recipient of that. Is this project not keeping parity with upstream?

**Reply by JulianPrieber** [reply]
> Is this project not keeping parity with upstream?

Mostly.
This is fixed now.

**Reply by beedell-roke** [reply]
> https://github.com/orgs/LinkStackOrg/discussions/407#discussioncomment-7246575

Thank you, @JulianPrieber. I'm glad to hear that.
